documentation/vscode_and_bolt.md

```diff
@@ -44,6 +45,9 @@ To start using the JSON schemas with VS Code, follow these steps:
         ],
         "https://forgeapi.puppet.com/schemas/bolt-inventory.json": [
           "inventory.yaml"
+        ],
+        "https://forgeapi.puppet.com/schemas/bolt-yaml-plan.json": [
```

Comment:
This is just cause I don't know: what's the process for publishing this here?

Reply:
It should be automatic as soon as it's merged to main.

Reply:
https://github.com/puppetlabs/puppet-forge-api/blob/main/app/controllers/schemas.rb#L41-L53

Here's the specific handler. It uses the filename from the request and locates the corresponding schema in the repo.

Reply:
Sweet! :tada: